substrate/frame/conviction-voting/src/lib.rs

```diff
@@ -68,9 +68,9 @@ pub type BlockNumberFor<T, I> =
 	<<T as Config<I>>::BlockNumberProvider as BlockNumberProvider>::BlockNumber;
 
 type AccountIdLookupOf<T> = <<T as frame_system::Config>::Lookup as StaticLookup>::Source;
-type BalanceOf<T, I = ()> =
+pub type BalanceOf<T, I = ()> =
 	<<T as Config<I>>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
-type VotingOf<T, I = ()> = Voting<
+pub type VotingOf<T, I = ()> = Voting<
```

Comment:
we also have some problem here with clone impl because of `MaxVotes`

Reply:
Yea I remember, I tried it but it introduces a lot of code since we have to manually implement Clone, Debug, PartialEq and Eq. Not really worth it in this case i think we can keep the aliases.